### PR TITLE
Proposed workflow changes

### PR DESCRIPTION
**Pull request by GuyHarwood**
please contribute and highlight any potential gaps

### CODE REVIEW

**Comment by GeorgeChatzigiannis** [comment]
Do we need an additional flow description for bug fixing?

**Comment by GuyHarwood** [comment]
@GeorgeChatzigiannis added steps for when feature rejected during sign off.

**Comment by GeorgeChatzigiannis** [comment]
@GuyHarwood you mention a feature branch shall be created from dev to fix issue.
Shouldn't it be a bug branch (under bug folder) instead?

**Comment by GeorgeChatzigiannis** [comment]
Actually forget my previous comment, it's still on dev so it is still development work.

**Comment by GuyHarwood** [comment]
@GeorgeChatzigiannis Exactly - the PBI is still in progress.  We dont want to create bugs for current work

File: deploy/workflow/workflow.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+![work-starts](https://github.com/DFEAGILEDEVOPS/MTC/blob/cd3898d3cba00fd73f791341152b8a1abd8479c8/deploy/workflow/work-starts.png)
+
+## Step 2: Work in progress for dev & QA
+
+![work-in-progress](https://github.com/DFEAGILEDEVOPS/MTC/blob/cd3898d3cba00fd73f791341152b8a1abd8479c8/deploy/workflow/work-in-progress-review.png)
+
+## Step 3: Dev & QA work complete
+
+![dev-qa-complete](https://github.com/DFEAGILEDEVOPS/MTC/blob/a45c8dbf90136695dec669bfe30f32280af93553/deploy/workflow/dev-qa-complete.png)
```
Comment (jon-shipley) [inline deploy/workflow/workflow.md]: My main concern with this is that unless we run the e2e tests before merging in to dev, dev will simply become a permanently broken branch, making it hard to work from.

We will also have to manually investigate test failures on the dev->master merge and guess which PR merge to dev caused the issue to assign it to someone.

The fact that this is a manual investigation is enough for me to not be in favour of it.

Have I missed something here?

Comment (GuyHarwood) [reply deploy/workflow/workflow.md]: `dev will simply become a permanently broken branch, making it hard to work from`  we _could_ create branches from master.  we should explore this idea.

`We will also have to manually investigate test failures on the dev->master merge` we are doing this anyway.

`... and guess which PR merge to dev caused the issue to assign it to someone.` it should be obvious.  QA and Dev should both have working tests before the merge to master happens.

**Pull request closed without merging**